samples/ibm-samples/ffdl-seldon/ffdl_pipeline.ipynb

```diff
@@ -0,0 +1,229 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# FfDL Kubeflow Pipeline Notebook demo\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define the necessary environment variables and install the KubeFlow Pipeline SDK\n",
+    "We assume this notebook kernel has access to Python's site-packages and is in Python3.\n",
+    "\n",
+    "**Please fill in the below environment variables with you own settings.**\n",
+    "\n",
+    "- **EXPERIMENT_NAME**: A unique experiment name that will be created for this notebook demo.\n",
+    "- **KFP_PACKAGE**: The latest release of kubeflow pipeline platform library.\n",
+    "- **KUBEFLOW_PIPELINE_LINK**: The link to access the KubeFlow pipeline API."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "EXPERIMENT_NAME = 'jupyter-demo'\n",
+    "KFP_PACKAGE = 'https://storage.googleapis.com/ml-pipeline/release/0.1.6/kfp.tar.gz'\n",
+    "KUBEFLOW_PIPELINE_LINK = 'http://169.48.201.216:30480/pipeline'"
```

Comment:
Does this IP change depending on the cluster? Is there a fixed DNS address that can be used?

Reply:
We would add a placeholder - that this needs to point to user`s KFP URL